File: rb/features.rb
# LasVegasCity SDK feature factory

require_relative 'feature/base_feature'
require_relative 'feature/test_feature'


module LasVegasCityFeatures
  def self.make_feature(name)
    case name
    when "base"
      LasVegasCityBaseFeature.new
    when "test"
      LasVegasCityTestFeature.new
    else
      LasVegasCityBaseFeature.new
    end
  end
end

File: py/features.py
# LasVegasCity SDK feature factory

from feature.base_feature import LasVegasCityBaseFeature
from feature.test_feature import LasVegasCityTestFeature


def _make_feature(name):
    features = {
        "base": lambda: LasVegasCityBaseFeature(),
        "test": lambda: LasVegasCityTestFeature(),
    }
    factory = features.get(name)
    if factory is not None:
        return factory()
    return features["base"]()
